### PR TITLE
PLATFORM-870: Set a timeout for maximum amount of scroll time

### PR DESCRIPTION
**Pull request by mjgp2**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -384,6 +385,11 @@ server.checkIfPageIsDoneLoading = function(req, res, force) {
             req.prerender.downloadFinished = new Date();
             // Do the loop that checks for infinite scrolling
             if(req.prerender.parameters['inf']) {
+
+                setTimeout(function() {
+                    scrollTimeout = true;
+                }, 30000);
```
Comment (mjgp2) [inline lib/server.js]: Can this be a constant as the pattern at the top of the file?

File: lib/server.js [hunk 2]
```diff
@@ -384,6 +385,11 @@ server.checkIfPageIsDoneLoading = function(req, res, force) {
             req.prerender.downloadFinished = new Date();
             // Do the loop that checks for infinite scrolling
             if(req.prerender.parameters['inf']) {
+
+                setTimeout(function() {
+                    scrollTimeout = true;
```
Comment (mjgp2) [inline lib/server.js]: Can we not just return here immediately?

Comment (Bam4d) [inline lib/server.js]: we still have to inject our JS and execute it (if we even use this functionality?) if pre-render is taking to long to do the scroll then the requests will time out inside webkit and return when they are done

File: lib/server.js (hunk 1 above)
Comment (Bam4d) [inline lib/server.js]: done

**Pull request merged**